modules/nats/options.go

```diff
@@ -49,3 +52,11 @@ func WithArgument(flag string, value string) CmdOption {
 		o.CmdArgs[flag] = value
 	}
 }
+
+// WithConfigFile pass io.Reader to the NATS container as /etc/nats.conf
+// Changes of a connectivity (listen address, or ports) may break a testcontainer
+func WithConfigFile(config io.Reader) CmdOption {
```

Comment:
suggestion: use a [ContainerCustomiser](https://pkg.go.dev/github.com/testcontainers/testcontainers-go#ContainerCustomizer) to avoid a two step process.

Reply:
https://github.com/testcontainers/testcontainers-go/pull/2905/commits/5da61731f5dfb3f77ab224f22ba36e3149307042